### DISCUSSION TITLE
Network management UI: add system proxy default TUN Mode

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [helloSystem/hello#561] [Ideas]
### Describe the feature or enhancement

* Network management UI: System proxy: TUN Mode [Wintun](https://github.com/nulldotblack/wintun)
[ligolo-ng](https://github.com/nicocha30/ligolo-ng)
[wireguard TUN](https://github.com/WireGuard/wireguard-go/tree/master/tun)
[Universal TUN/TAP driver](https://vtun.info/)
[tun2socks](https://github.com/xjasonlyu/tun2socks)
[tun2proxy](https://github.com/blechschmidt/tun2proxy)

https://man.freebsd.org/cgi/man.cgi?query=tun
Network management UI added option TUN 
TUN acts as an agent for existing agreements, HTTP proxy HTTPS proxy SOCKS proxy
Use TUN mode to proxy all traffic throughout the system, including terminals

### Why this request?

I’m in a country where the internet is very blocked, and upgrading or downloading software is a hassle, github gitlab Ping over time 300ms.

In thousands of Linux and BSD distributions, there is no single system desktop environment that proxies all traffic,
I use a proxychains and it’s a terrible proxy for the terminal, very slow.

**Comment by probonopd** [comment]
Thanks for suggesting to add an easy way to tunnel/proxy all traffic.

We would probably need to write a small GUI tool (in PyQt5) to configure it.
